### ISSUE TITLE
Strange argument error starting in 1.5.x

### ISSUE THREAD

**Issue by kemiller** [HipByte/motion-yaml#11]
```
<Warning>: yaml.rb:28:in `load:': wrong number of arguments (-1685773368 for 0) (ArgumentError)
<Warning>: yaml.rb:28:in `load:': wrong number of arguments (-1685773368 for 0) (ArgumentError)
<Error>: *** Terminating app due to uncaught exception 'ArgumentError', reason: 'yaml.rb:28:in `load:': wrong number of arguments (-1685773368 for 0) (ArgumentError)
```

1.4 does not have this issue, and it only shows up on the device. 

See here for context (starting with 2nd comment):  https://github.com/infinitered/cdq/issues/105

**Comment by Watson1978** [comment]
I reproduced the problem.
Looks like It will be occurred on ARM64 device only.
So, you can use a workaround as following until that it will be solved.

```ruby
Motion::Project::App.setup do |app|
  ...
  app.archs['iPhoneOS'] = ['armv7']
end
```

**Comment by Watson1978** [comment]
I think this issue is reproduced since https://github.com/HipByte/motion-yaml/commit/eca66dac3ce428d1a3c29a2f330cb46a1ae73205

**Comment by Watson1978** [comment]
I filed the issue in http://hipbyte.myjetbrains.com/youtrack/issue/RM-906

**Comment by Watson1978** [comment]
This issue should be solved with motion-yaml 1.5.3.

Thank you for reporting!
